File: Submissions/T06/M1.py
nilai = input("Masukkan 5 nilai : ").split()

if len(nilai) != 5:
    print("Harus ada 5 nilai")
else:
    total = 0

    for i in nilai:
        total += int(i)
    rata = total / 5
    print(f"Rata-rata : {rata: .2f}")

    if rata >= 80:
        grade = "A"
    elif rata >= 70:
        grade = "B"
    elif rata >= 60:
        grade = "C"
    elif rata >= 50:
        grade = "D"
    else:
        grade = "E"

    print(f"Grade : {grade}")
    

File: Submissions/T06/M1.cpp
#include <stdio.h>

int main(){
	
	float nilai[5];
	float r = 0;
	

	printf("Masukkan 5 nilai: ");
	for (int i = 0; i < 5; i++){
		scanf("%f", &nilai[i]);
	}
	

	for (int i = 0; i < 5; i++){
		r += nilai[i];
	}
	r = r / 5;
	printf("\nRata-rata: %.2f\n", r);
	
	
	if (r >= 80){
		printf("Grade: A\n");
	} else if (r >= 70){
		printf("Grade: B\n");
	} else if (r >= 60){
		printf("Grade: C\n");
	} else {
		printf("Grade: D\n");
	}
	
	return 0;
}
